cmd/evm/internal/t8ntool/flags.go

```diff
@@ -28,12 +28,15 @@ import (
 var (
 	TraceFlag = &cli.BoolFlag{
 		Name:  "trace",
-		Usage: "Output full trace logs to files <txhash>.jsonl",
+		Usage: "Output full trace logs to a file named: trace-<txIndex>-<txHash>.json or trace-<txIndex>-<txHash>.jsonl",
```

Comment:
```suggestion
		Usage: "Output json opcode traces to files (trace-<txIndex>-<txHash>.jsonl),
```
IMO the `--trace` configures the json tracer. For the other ones, it's implied. Why would you want to run a calltracer if it does not produce any output? 

I'll push a fix